arviz/plots/loopitplot.py

```diff
@@ -210,14 +210,14 @@ def plot_loo_pit(
 
         unif = np.random.uniform(size=(n_unif, loo_pit.size))
         x_vals = np.linspace(0, 1, len(loo_pit_kde))
-        if use_hpd:
-            if hpd_kwargs is None:
-                hpd_kwargs = {}
-            hpd_kwargs.setdefault("color", to_hex(hsv_to_rgb(light_color)))
-            hpd_fill_kwargs = hpd_kwargs.pop("fill_kwargs", {})
-            hpd_fill_kwargs.setdefault("label", "Uniform HPD")
-            hpd_kwargs["fill_kwargs"] = hpd_fill_kwargs
-            hpd_kwargs["credible_interval"] = credible_interval
+        if use_hdi:
```

Comment:
for loo pit I left it hpd because this particular case it is hpd. I don't think loo pit makes sense for anything bu the posterior. Not opposed to changing it but wanted to get your thoughts

Reply:
that's true, but having hpd and hdi seems to me like a source of potential confusion for users or even ourselves in the future